package.json

```diff
@@ -19,21 +19,17 @@
   "scripts": {
     "test": "make test",
     "start": "electron lib/start.js",
-    "preshrinkwrap": "node ./scripts/clean-shrinkwrap.js",
+    "postshrinkwrap": "node ./scripts/clean-shrinkwrap.js",
     "configure": "node-gyp configure",
     "build": "node-gyp build",
     "install": "node-gyp rebuild"
   },
   "author": "Resin Inc. <hello@etcher.io>",
   "license": "Apache-2.0",
-  "shrinkwrapIgnore": [
-    "macos-alias",
-    "fs-xattr",
-    "ds-store",
-    "appdmg",
-    "7zip-bin-mac",
-    "7zip-bin-win",
-    "7zip-bin-linux"
+  "platformSpecificDependencies": [
+    [ "7zip-bin-mac" ],
+    [ "7zip-bin-win" ],
+    [ "7zip-bin-linux" ]
```

Comment:
The new approach works based on shrinkwrap "paths", which we set using array syntax. Basically this means that we want to omit `7zip-bin-mac` from the top level of the shrinkwrap file for example. If `7zip-bin-mac` was present inside `foo` in the shrinkwrap file, we should add `[ 'foo', '7zip-bin-mac' ]`.